style.css

```diff
@@ -42,3 +42,10 @@
     position: relative;
     top: 2px;
 }
+
+/* Ugly hack to align search button.
+ * Without this, the search button is slightly to the top
+ * of the input field and the refresh button. */
+label[for="main-search-expandable"] {
```

Comment:
eh, I'd give the thing a class instead of using label, but if you think this is robust enough, sure...